test/py/util.py

```diff
@@ -28,5 +28,6 @@
 def local_libpath():
     globdir = os.path.join(_ROOT_DIR, 'lib', 'py', 'build', 'lib.*')
     for libpath in glob.glob(globdir):
-        if libpath.endswith('-%d.%d' % (sys.version_info[0], sys.version_info[1])):
+        if libpath.endswith('-%d.%d' % (sys.version_info[0], sys.version_info[1])) \
+            or libpath.endswith('-%s' % sys.implementation.cache_tag):
```

Comment:
can you add inline code comment explaining this?

Reply:
Done